File: src/test/java/com/parmet/squashlambdas/cal/CalUtilsTest.java
package com.parmet.squashlambdas.cal;

import com.parmet.squashlambdas.testutil.ConfiguredTest;
import org.junit.Ignore;
import org.junit.Test;

public class CalUtilsTest extends ConfiguredTest {
  @Test
  @Ignore
  public void giveUserOwnership() throws Exception {
    CalUtils.printAcl(calendar(), config().getString("google.calendarId"));
    CalUtils.giveUserOwnership(calendar(), config().getString("google.calendarId"), "");
    CalUtils.printAcl(calendar(), config().getString("google.calendarId"));
  }
}

File: src/test/kotlin/com/parmet/squashlambdas/cal/CalUtilsTest.kt
package com.parmet.squashlambdas.cal

import com.parmet.squashlambdas.testutil.ConfiguredTest
import org.junit.jupiter.api.Disabled
import org.junit.jupiter.api.Test

class CalUtilsTest : ConfiguredTest() {
    @Test
    @Disabled
    fun `give a new user ownership of the config's calendarId`() {
        printAcl(calendar, emailNotificationConfig.googleCal.calendarId)
        giveUserOwnership(calendar, emailNotificationConfig.googleCal.calendarId, "")
        printAcl(calendar, emailNotificationConfig.googleCal.calendarId)
    }
}
